nccs/pipeline/direct/agriculture.py

```diff
@@ -27,10 +69,15 @@ def get_exposure(crop_type: CropType = "whe", scenario="histsoc", irr: Irrigatio
     )
 
 
-def get_impf_set():
+def get_impf_set(crop_type: typing.Union[CropType, None] = None):
+    # TODO: check if other impact functions are needed
     impf_cp = ImpactFuncSet()
     impf_def = ImpfRelativeCropyield()
     impf_def.set_relativeyield()
+
+    # Invert the impact function to match the expected behavior
+    impf_def.intensity = impf_def.intensity
```

Comment:
Shoud this rather be
```suggestion
    impf_def.mdd = -impf_def.mdd
```